extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -118,13 +120,74 @@
   transform-origin: right !important;
 }
 
+[info] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: 48px !important; 
+}
+
+[info] [dir=rtl] .i-amphtml-message-container {
```

Comment:
I believe this selector doesn't work since `dir=rtl` and `info` are set on the same element.
It should be `[info][dir=rtl]`, without a space between both selectors.